app/services/data/audit/get-all-claims-data-below-threshold.js

```diff
@@ -0,0 +1,22 @@
+const moment = require('moment')
+const {
+  getDatabaseConnector
+} = require('../../../databaseConnector')
+
+module.exports = function (startDate, endDate, percent, threshold) {
+  const db = getDatabaseConnector()
+  return db('Claim')
+    .select('Reference')
```

Comment:
You can do `    .select('Reference', 'ClaimId', 'PaymentAmount')`

Reply:
> You can do ` .select('Reference', 'ClaimId', 'PaymentAmount')`

Updated